manifests/charts/istio-control/istio-discovery/files/kube-gateway.yaml

```diff
@@ -76,6 +76,10 @@ spec:
         - name: net.ipv4.ip_unprivileged_port_start
           value: "0"
       {{- end }}
+      {{- if .Values.gateways.seccompProfile }}
```

Comment:
Do we want this within the `else` on L74? Basically either you can set securityContext or seccompProfile.

Or should we just require them to use securityContext entirely for seccompProfile?

Reply:
Yep, it makes more sense to put it in the `else` clause. Otherwise someone supplying their own could cause duplicate key collisions (which Helm would report during templating and that person could resolve easily enough).